staging/src/k8s.io/apiserver/pkg/cel/library/cost_test.go

```diff
@@ -1191,6 +1191,70 @@ func TestSemverCost(t *testing.T) {
 	}
 }
 
+func TestImageCost(t *testing.T) {
+	cases := []struct {
+		name                string
+		expr                string
+		expectEstimatedCost checker.CostEstimate
+		expectRuntimeCost   uint64
+	}{
+		{
+			name:                "parse",
+			expr:                `image("reg.io/repo/img:tag")`,
+			expectEstimatedCost: checker.CostEstimate{Min: 2, Max: 2},
+			expectRuntimeCost:   2,
+		},
+		{
+			name:                "isImage",
+			expr:                `isImage("reg.io/repo/img:tag")`,
+			expectEstimatedCost: checker.CostEstimate{Min: 2, Max: 2},
+			expectRuntimeCost:   2,
+		},
+		{
+			name:                "containsDigest",
+			expr:                `image("reg.io/repo/img@sha256:6aefddb645ee6963afd681b1845c661d0ea4c3b20ab9db86d9e753b203d385f2").containsDigest()`,
```

Comment:
Probably at least one of these should cover `:tag@digest`

We should also probably cover the non-sha256 digests.

Reply:
I have added those in [staging/src/k8s.io/apiserver/pkg/cel/library/image_test.go](https://github.com/kubernetes/kubernetes/pull/130834/files#diff-bb40882d35c8fe09c47256f825df4923942951f7d188249dc4e3aa2e360cead6R216-R220). 
I didn't add them to cost test, should I add it there too?